### DISCUSSION TITLE
Is CM3 safe to use?

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [modula3/cm3#1135] [General]
Such a lengthy thread: https://github.com/modula3/cm3/discussions/1128 but even after I read the whole of it, I'm still not sure. I'm not a lawyer and I don't understand the words of the various COPYRIGHT files on the root of the repository. Please tell me, if I suggested CM3 to someone and he developed his commercial software in CM3, will suddenly a day his door is knocked and he got sued or not allowed to sell his software anymore because violation of some obscure licensing issues? This is the only thing I wanted to know. I don't want to redistribute CM3 or sell it so I don't care about problems regarding this.

**Comment by olaf-wagner** [comment]
On 6 Jan 2023, at 20:47, nsgnkhibdk2cls0f wrote:

> Such a lengthy thread: https://github.com/modula3/cm3/discussions/1128 
> but even after I read the whole of it, I'm still not sure. I'm not a 
> lawyer and I don't understand the words of the various COPYRIGHT files 
> on the root of the repository. Please tell me, if I suggested CM3 to 
> someone and he developed his commercial software in CM3, will suddenly 
> a day his door is knocked and he got sued or not allowed to sell his 
> software anymore because violation of some obscure licensing issues? 
> This is the only thing I wanted to know. I don't want to redistribute 
> CM3 or sell it so I don't care about problems regarding this.

You always could and can use any Module-3 compiler (DEC SRC, PM3, CM3)
to compile your own software and distribute and sell it without any 
risk.
That said, I am not a lawyer, too, so you might not believe me.

> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/modula3/cm3/discussions/1135
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>
-- 
Olaf Wagner -- elego Software Solutions GmbH -- http://www.elegosoft.com
                Gustav-Meyer-Allee 25 / Gebäude 12, 13355 Berlin, 
Germany
Geschäftsführer: Olaf Wagner | Sitz: Berlin
Handelregister: Amtsgericht Charlottenburg HRB 77719 | USt-IdNr: 
DE163214194
